### DISCUSSION TITLE
Sometimes script fails to open class

### DISCUSSION THREAD

**Discussion by skyshines** [TobiasPankner/Teams-Auto-Joiner#170] [General]
when the bot opens calender [mode - 3 calender only]
and the class goes in pop up
after that it should come back to class

but many a times it shows this
[10:23:24] Looking for new meetings
Found meetings: 
	chem lecture_ extra from 09:50 to 10:50 1642652400 [Calendar]

[10:23:37] Looking for new meetings
Found meetings: 
	chem lecture_ extra from 09:50 to 10:50 1642652400 [Calendar]
Failed to open meeting member page

[10:23:50] Looking for new meetings
Found meetings: 
	chem lecture_ extra from 09:50 to 10:50 1642652400 [Calendar]
Failed to open meeting member page

[10:24:02] Looking for new meetings
Found meetings: 
	chem lecture_ extra from 09:50 to 10:50 1642652400 [Calendar]
Failed to open meeting member page

and that too repeatedly the max
any solution?
should i update something

**Comment by skyshines** [comment]
@TobiasPankner pls look into it when free

**Comment by TobiasPankner** [comment]
What can you see in the browser window when the error occurs? Is the meeting in fullscreen?

**Reply by skyshines** [reply]
first meeting is opened full screen -> then it goes to calender to check for new meetings and the current meeting becomes a pop up-> since no new class tries to go back to the meeting currently in pop up but couldnt

**Reply by skyshines** [reply]
and shows the error shows up
so in short
cant openback meeting in full from pop up in calender

**Reply by skyshines** [reply]
@TobiasPankner

**Reply by TobiasPankner** [reply]
I cannot really fix this because I don't have a paid Teams account with the calendar enabled. If you know how, you are welcome to fix it and submit a Pull Request

**Reply by skyshines** [reply]
![Screenshot at 2022-01-20 12-46-32](https://user-images.githubusercontent.com/87031909/150938759-33213869-c48d-4bcc-ac56-705349588ce5.png)

**Reply by skyshines** [reply]
@TobiasPankner

**Reply by skyshines** [reply]
Sir as you can see continous failed meeting member page error
the meeting is in the pop up

**Reply by skyshines** [reply]
@TobiasPankner i am sorry to tag again but can you look at this when free

**Reply by TobiasPankner** [reply]
> I cannot really fix this because I don't have a paid Teams account with the calendar enabled. If you know how, you are welcome to fix it and submit a Pull Request

What should I look at? As said above I cannot fix this. Unless you send me the css selectors of the popup window if you know how to do that.

**Reply by TobiasPankner** [reply]
I can't just look at the picture and write code to click that

**Reply by skyshines** [reply]
i am sorry
Thanks for making the bot
and I tried only channel mode
it is working enough for me

I will read about CSS selectors and get back to you
